AMW_angular/io/src/app/resources/resources-page.component.html

```diff
@@ -30,6 +30,13 @@
             }
           </nav>
         }
+        <div>
```

Comment:
Move the "New Resource Type"-Button above the root resource types:

![IMG_20241203_131019](https://github.com/user-attachments/assets/3e0a45f1-dcba-417d-bd1b-7b105dd05804)